Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -357,8 +353,11 @@ void SpreadsheetWidget::setup_tabs(NonnullRefPtrVector<Sheet> new_sheets)
     if (first_tab_widget)
         change(*first_tab_widget);
 
-    m_tab_widget->on_change = [change = move(change)](auto& selected_widget) {
+    m_tab_widget->on_change = [change = move(change), this](auto& selected_widget) {
         change(selected_widget);
+
+        // for keyboard shortcuts and command palette
+        m_tab_context_menu_sheet_view = static_cast<SpreadsheetView&>(selected_widget);
```

Comment:
Is it possible to set `m_tab_context_menu_sheet_view ` inside `change()` ?

Reply:
yeah, it is possible, but I don’t see a reason why we should move it there honestly.

It looks like the `change()` lambda function just moves function events from one view to another, so it looks more like a setup function (that probably should be improved) so setting the current tab there for the context menu doesn’t seem right.

Reply:
a small update: The `change()` function no longer exists (see #13032). Things from that function are now set immediately when making a tab.